Assets/scripts/LoadCatalog.cs

```diff
@@ -275,6 +273,52 @@ private void ChangeContentToCategory(string category)
         }        
     }
 
+    //Generates Items listed under Office category
+    public void categoryOffice()
+    {
+        GameObject content = GameObject.Find("Content");
+
+        foreach (Item itemInList in loadedItems)
+        {
+            if (itemInList.GetCategories().Contains("Office"))
+            {
+                showListing(itemInList);
+                content.transform.Find($"Listing: {itemInList.GetItemID()} {itemInList.GetName()}").gameObject.SetActive(true);
+            }
+        }
+    }
+
+    //Generates Items listed under Living Category
+    public void categoryLiving()
+    {
+        GameObject content = GameObject.Find("Content");
+
+        foreach (Item itemInList in loadedItems)
+        {
+            if (itemInList.GetCategories().Contains("Living Room"))
+            {
+                showListing(itemInList);
+                content.transform.Find($"Listing: {itemInList.GetItemID()} {itemInList.GetName()}").gameObject.SetActive(true);
+            }
+        }
+    }
+
+    //Generates Items listed under Google category
+    public void categoryGoogle()
+    {
+        GameObject content = GameObject.Find("Content");
+
+        foreach (Item itemInList in loadedItems)
+        {
+            if (itemInList.GetCategories().Contains("Google"))
+            {
+                showListing(itemInList);
+                content.transform.Find($"Listing: {itemInList.GetItemID()} {itemInList.GetName()}").gameObject.SetActive(true);
+            }
+        }
+    }
+
+
```

Comment:
These methods are not needed as the ChangeContentToCategory method takes in a string parameter (such as "Google", "Office", or "Living Room"). For example: calling categoryGoogle() will do the exact same thing as ChangeContentToCategory("Google")

Try and make it so when you click on a filter it will call the ChangeContentToCategory() method with the category string in the parameters.

Reply:
Alright, I'll trim this down.

Reply:
Alright I'll trim them all down. Thanks for the feedback. Will keep you guys updated :)